test/cql-pytest/cassandra_tests/validation/operations/select_single_column_relation_test.py

```diff
@@ -522,7 +522,7 @@ def testInvalidSliceRestrictionOnPartitionKey(cql, test_keyspace):
         # ALLOW FILTERING, while Scylla *also* tells you that this
         # query would have worked with EQ or IN relations or with token().
         # The word "filtering" is common to both messages.
-        assert_invalid_message(cql, table, 'filtering',
+        assert_invalid_message(cql, table, 'FILTERING',
```

Comment:
Ditto. I think this will break the test on Cassandra.
I basically have the same question in many tests below, I'll not list them all.